src/client/camera.cpp

```diff
@@ -718,7 +722,8 @@ void Camera::drawNametags()
 			// shadow can remain.
 			continue;
 		}
-		v3f pos = nametag_pos + nametag->nametag_pos * BS;
+		v3f pos = nametag->parent_node->getAbsolutePosition() +
+			  nametag->nametag_pos * BS;
```

Comment:
We use one tab indents for continuation lines. We don't use spaces for alignment here.

```suggestion
			nametag->nametag_pos * BS;
```

Reply:
Whoops